Crash consistent VM restore points (preview)/createRestorePoint.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "RPCName": {
+      "type": "string",
+      "defaultValue": "<RestorePointCollectionName>"
+    },
+    "RPCResourceId": {
+      "defaultValue": "/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroupName>/providers/Microsoft.Compute/restorePointCollections/<RestorePointCollectionName>",
+      "type": "String"
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Compute/restorePointCollections/restorePoints",
+      "name": "[concat( parameters('RPCName'),'/rp1')]",
```

Comment:
Can we change this to  "name": "[concat( parameters('RPCName'),'/rp1')]",

Like the one for normal restore points currently we have.

Reply:
@prchin , what is the difference between what you are mentioning and what I have put? I dont see any difference.

Reply:
Sorry, my bad. I meant something like this "[concat(parameters('RPCName'),'/', parameters('RPName'))]"

Reply:
Got it. Done.